### ISSUE TITLE
sqlite3.OperationalError

### ISSUE THREAD

**Issue by Vykq** [ArtVentureX/sd-webui-agent-scheduler#203]
Sometimes I get this error. It can be no matter what API request is made, i get it randomly.

{
    "error": "OperationalError",
    "detail": [],
    "body": "",
    "errors": "(sqlite3.OperationalError) database is locked\n[SQL: SELECT task.id AS task_id, task.api_task_id AS task_api_task_id, task.api_task_callback AS task_api_task_callback, task.name AS task_name, task.type AS task_type, task.params AS task_params, task.script_params AS task_script_params, task.priority AS task_priority, task.status AS task_status, task.result AS task_result, task.bookmarked AS task_bookmarked, task.created_at AS task_created_at, task.updated_at AS task_updated_at \nFROM task \nWHERE task.id = ?]\n[parameters: ('1b8d1343-6aeb-4802-a40f-b2b0e60b696e',)]\n(Background on this error at: https://sqlalche.me/e/20/e3q8)"
}

**Comment by mcangeli** [comment]
Yep, I'm getting it as well, haven't been able to tell why or what the cause is.
